### ISSUE TITLE
No Remove Member functionality

### ISSUE THREAD

**Issue by calvin-shirley** [acm-uiuc/liquid#85]
There's currently no way to remove a user from a group via the group manager.  There should be.  

Marking them as frozen will remove them from displaying on the SIG/committee page, but will still tell the user that they're a member of the group when they log into intranet.  Also, over time, the list will grow to become very large if there isn't a way to remove members that are no longer members from the group.

tl;dr somebody implement "remove user from group."
